File: CS/MyXtraGrid/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace MyXtraGrid
{
    public class Customer
    {
        string name;
        int age;
        int weight;
        int height;

        public string Name
        {
            get { return name; }
            set { name = value; }
        }
        public int Age
        {
            get { return age; }
            set { age = value; }
        }
        public int Weight
        {
            get { return weight; }
            set { weight = value; }
        }
        public int Height
        {
            get { return height; }
            set { height = value; }
        }
    }
}

File: VB/MyXtraGrid/Customer.vb
Namespace MyXtraGrid

    Public Class Customer

        Private nameField As String

        Private ageField As Integer

        Private weightField As Integer

        Private heightField As Integer

        Public Property Name As String
            Get
                Return nameField
            End Get

            Set(ByVal value As String)
                nameField = value
            End Set
        End Property

        Public Property Age As Integer
            Get
                Return ageField
            End Get

            Set(ByVal value As Integer)
                ageField = value
            End Set
        End Property

        Public Property Weight As Integer
            Get
                Return weightField
            End Get

            Set(ByVal value As Integer)
                weightField = value
            End Set
        End Property

        Public Property Height As Integer
            Get
                Return heightField
            End Get

            Set(ByVal value As Integer)
                heightField = value
            End Set
        End Property
    End Class
End Namespace
